meson.build

```diff
@@ -1,7 +1,7 @@
 project('fwupd', 'c',
   version : '1.6.0',
   license : 'LGPL-2.1+',
-  meson_version : '>=0.47.0',
+  meson_version : '>=0.53.0',
```

Comment:
can avoid this change if doing the dependency checking stuff.

Reply:
With a RHEL head, that would indeed be good to avoid.